### PR TITLE
Add attribute missing logic for Constraints

### PR DESCRIPTION
**Pull request by iven**
See #1102

### CODE REVIEW

File: src/main/scala/mesosphere/mesos/Constraints.scala [hunk 1]
```diff
@@ -108,6 +108,13 @@ object Constraints {
       }
     }
 
+    private def checkMissingAttribute = {
+      constraint.getOperator match {
```
Comment (drexin) [inline src/main/scala/mesosphere/mesos/Constraints.scala]: This should be `constraint.getOperator == Operator.UNLIKE`

**Comment by drexin** [comment]
Thanks! If you change the match to a comparison I will merge this.

**Comment by iven** [comment]
@drexin Like this?

**Comment by drexin** [comment]
@iven Yes, if you also remove the parentheses, I will merge it. Thanks!

**Comment by iven** [comment]
@drexin Done.

**Comment by drexin** [comment]
:100: Thanks a lot!

**Pull request merged**